includes/free/class-login.php

```diff
@@ -1091,4 +1079,64 @@ public function default_wp_login_override() {
             exit;
         }
     }
+
+    /**
+     * Send activated email for user
+     *
+     * @param $user
+     */
+    private function send_activated_email( $user ) {
+        $blogname   = wp_specialchars_decode( get_option( 'blogname' ), ENT_QUOTES );
+        $user_email = $user->user_email;
+        $to         = $user_email;
```

Comment:
You don't need to declare variables if you don't need to. Both `$user_email` and `$to` variables aren't needed. Use `$user->user_email` directly.